plugins/woocommerce/client/admin/client/settings-email/settings-email-preview-slotfill.tsx

```diff
@@ -34,6 +34,13 @@ const EmailPreviewFill: React.FC< EmailPreviewFillProps > = ( {
 	const [ emailType, setEmailType ] = useState< string >(
 		'WC_Email_Customer_Processing_Order'
 	);
+	const [ finalPreviewUrl, setFinalPreviewUrl ] =
+		useState< string >( previewUrl );
+
+	useEffect( () => {
+		setFinalPreviewUrl( `${ previewUrl }&type=${ emailType }` );
+	}, [ emailType, previewUrl ] );
+
```

Comment:
Is `useEffect` necessary here? Looks like this whole chunk can be simplified to just an assignment:
```
const finalPreviewUrl = `${previewUrl}&type=${emailType}`;
```

Reply:
Good catch! You're right that there is no need for `useEffect` in this case.  Addressed in 0e815ca.